autodepgraph/svg_viewer/adg_graph.svg

```diff
@@ -1,63 +1,25 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN"
  "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
-<!-- Generated by graphviz version 2.38.0 (20140413.2041)
+<!-- Generated by graphviz version 2.41.20180302.1211 (20180302.1211)
```

Comment:
Maybe we should remove this file and add it to the gitignore...

Reply:
Yes, I'll do that